### ISSUE TITLE
Can not `setup-resources` in `v9.8.0`

### ISSUE THREAD

**Issue by wiktor-obrebski** [commercetools/commercetools-adyen-integration#965]
`setup-resources` script fail for me in build `commercetools/commercetools-adyen-integration-extension:v9.8.0`.
It worked with the same setup before.
Example below are simplified one, but it behave in the same way in my real scenario.

```sh
docker run -it commercetools/commercetools-adyen-integration-extension:v9.8.0 sh
npm run setup-resources

> commercetools-adyen-integration-extension@9.8.0 setup-resources
> node -e 'require("./src/setup.js").setupExtensionResources()'

node:internal/modules/cjs/loader:1156
      throw err;
      ^

Error [ERR_REQUIRE_ESM]: require() of ES Module /app/src/setup.js from /app/[eval] not supported.
Instead change the require of setup.js in /app/[eval] to a dynamic import() which is available in all CommonJS modules.
    at [eval]:1:1
    at Script.runInThisContext (node:vm:129:12)
    at Object.runInThisContext (node:vm:305:38)
    at [eval]-wrapper:6:22 {
  code: 'ERR_REQUIRE_ESM'
}
```

**Comment by praveenkumarct** [comment]
@wiktor-obrebski Thank you writing us the issue. I am currently fixing the issues on v9.8.0 extension release. We will be providing you with the fixes today in the latest version.

**Comment by praveenkumarct** [comment]
@wiktor-obrebski new release has been published and v9.8.1 version is available in docker also.

https://github.com/commercetools/commercetools-adyen-integration/releases/tag/v9.8.1

I am closing this issue, Thank you.
